client/src/pages/Home/Home.styles.ts

```diff
@@ -1,6 +1,96 @@
 import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { device } from "../../utils/Breakpoints";
+
+interface StyledProps {
+  percentage: number;
+}
 
 export const Container = styled.div`
   width: 100%;
-  padding: 2em;
+  max-width: 800px;
+  padding: 1em;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  gap: 2em;
+  align-items: center;
+`;
+
+export const PageTitle = styled.h3`
+  text-align: center;
+  text-transform: uppercase;
+  letter-spacing: 0.2em;
+  margin-top: 1em;
+
+  @media ${device.desktop} {
+    font-size: 2rem;
+  }
+`;
+
+export const BudgetMainContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5em;
+  box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+  padding: 1em;
+  border-radius: 5px;
+`;
+
+export const BudgetTitle = styled.h1`
+  letter-spacing: 0.1em;
+  font-weight: 800;
+  text-transform: uppercase;
+`;
+
+export const Soft = styled.span`
+  font-weight: 400;
+`;
+
+export const TotalBudgetBar = styled.div`
+  width: 100%;
+  height: 30px;
+  background: lightgray;
+  border-radius: 5px;
+  position: relative;
+`;
+
+export const ExpenseBar = styled.div<StyledProps>`
```

Comment:
You don't have to, but there is an html element called <progress> which is legit progress bar. Not sure if you wanna use it or not.

Reply:
I looked into the progress - and it seems easy to implement but hard to style because I'll have to go into webkits. If it's better for performance, I can definitely research and get it done, but if this works okay for performance, I'll stick to it